### ISSUE TITLE
Double space on end of line is not always translated to <br>

### ISSUE THREAD

**Issue by mnn** [CONNECT-platform/codedoc#14]
Input (2 spaces after colon on a first line):
```md
**Type parameters**:  
`U`: `any` Output [GGArray](#GGArray) item type
```
is rendered in a browser as:
```html
<p><strong>Type parameters</strong>:  
<code>U</code>: <code>any</code> Output <a href="#GGArray">GGArray</a> item type</p>
```
It looks to me like those 2 spaces are outputted verbatim.

<img width="294" alt="2020-06-07_09-29" src="https://user-images.githubusercontent.com/1975567/83962865-738c5880-a8a1-11ea-91f6-97472aa80fd3.png">

On the project docs is mentioned that CommonMark is supported, I think this is the relevant part: https://spec.commonmark.org/0.29/#example-630

Interestingly, in some other cases it does work:

```md
**Parameters**:   
`f`: `FuncLike<T, U>` Mapping function  
`ctx`: `any` Context for function
```
Second hard line break is translated to `<br>`:
<img width="282" alt="2020-06-07_09-32" src="https://user-images.githubusercontent.com/1975567/83962926-00cfad00-a8a2-11ea-856e-adb1af4b72a9.png">

**Comment by loreanvictor** [comment]
This is an issue with `markedjs`. I have opened [an issue on that project](https://github.com/markedjs/marked/issues/1699), and will keep this open to monitor the status of that issue.

More information: the issue seems to happen when you have styled text (_emphasize_, **strong**), followed by a colon (`:`), followed by two (or more) spaces, does not result in a hard-line-break using `markedjs`. For a work-around, you can for example add `&puncsp;` after the colon or between the styled text and the colon.

**Comment by mnn** [comment]
I found `\` at the end of a line also works. It's not a critical issue, I have access to the markdown generator, so I'll later try modifying it. But in the long run I would prefer having two spaces instead of `\` or `&puncsp;`, because in my opinion those spaces followed by a new line are more readable in raw text form.

**Comment by loreanvictor** [comment]
Yes and regardless of readability compliance with common mark is essential. But I suspect it actually wouldn't be long until it is [fixed on Marked](https://github.com/markedjs/marked/issues/1699).
